locales/en.ts

```diff
@@ -392,5 +392,9 @@ const en: { [key: string]: string } = {
     supportLoaderSubText: `We’re fetching support details for you`,
     orderCancellation: 'Order Cancellation',
     cancellationType: 'Please select the reason for cancellation',
+    orderCancel: 'Order Cancelled!',
```

Comment:
Can you please add these in fa.ts also? You can translate these into french from google translate

Reply:
Done..